src/docs/user/references/ApplicationConnectors.adoc

```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
+
+===== Send Emails
+
+This workflow enables to send emails through ServiceNow using the EMAIL API.
+
+It is made of a single task with its variables.
+
+*Task Variables:*
+
+.ServiceNow_Send_Email
+[cols="2,5,2,2,2"]
+|===
+| *Variable name* | *Description* | *Required?*  | *Type*  | *Default/Examples*
+| `ENDPOINT`
+| ServiceNow server endpoint.
+| Yes
+| URL
+| `https://SN_SERVER/api/now/v1/email`
+| `USER`
+| ServiceNow user.
+| Yes
+| String
+| e.g. administrator
+| `PASSWORD`
+| ServiceNow password.
+| Yes
+| Sring (HIDDEN)
+| e.g. PWD
+| `TO`
+| Recipients of the email.
+| Yes
+| String as a comma separated list
+| e.g. `example@emailexample.com,example2@emailexample.com`
+| `CC`
+| Email addresses that will be in copy.
+| No
+| String as a comma separated list
+| e.g. `example@emailexample.com,example2@emailexample.com`
+| `SUBJECT`
+| The subject of the email.
+| No
+| String
+| e.g. `Email from ProActive about ...`
+| `HTML`
+| Content of the email, can contain HTML tags
+| Yes
+| String with HTML
+| e.g. `<bold> This content has been generated by proactive </bold>`
+| `TABLE_NAME`
+| Name of the table to save the email. Use this parameter to associate an email message to a particular related record elsewhere in the system.
+| No
+| String with HTML
+| e.g. incident
+| `TABLE_RECORD_ID`
+| Target-related record to which the email applies. Use this parameter to associate an email message to a particular related record elsewhere in the system.
+| No
+| String
+| e.g. 9d385017c611228701d22104cc95c371
+| `SSL_DISABLE_CHECK`
+| If true, disables SSL certificate verification
+| No
+| Boolean
+| e.g. true
+| `CONTENT_TYPE`
+| Content-type of the request, should be 'application/json' with eventual encoding.
+| Yes
+| String
+| e.g. `application/json;charset = UTF-8`
+| `REQUEST_BODY`
+| The body of the HTTP request which is automatically built.
+| Yes
+| String
+| e.g. `{"to":"","cc":"","subject":"${SUBJECT}","html":"${HTML}"}`
+| `DEBUG`
+| If true, activates the debug mode which prints messages to the terminal.
+| No
+| Boolean
+| e.g. `true`
+|===
+
+===== Table API
+
+The TABLE API allows to query data from a ServiceNow instance.
+
+There are six workflows in ProActive's Catalog that queries ServiceNow's TABLE API. Two enables to create a ServiceNow task
+and mark it as resolved and four other to perform basic CRUD operations on ServiceNow's tables.
+
+The two Task related workflows have both one task with the following variables:
+
+.ServiceNow_Create_Task
+[cols="2,5,2,2,2"]
+|===
+| *Variable name* | *Description* | *Required?*  | *Type*  | *Default/Examples*
+| `ENDPOINT`
+| ServiceNow server endpoint.
+| Yes
+| URL
+| `https://SN_SERVER/api/now`
+| `USER`
+| ServiceNow user.
+| Yes
+| String
+| e.g. administrator
+| `PASSWORD`
+| ServiceNow password.
+| Yes
+| Sring (HIDDEN)
+| e.g. PWD
+| `TASK_TYPE`
+| The subtype of the ServiceNow Task
+| Yes
+| String
```

Comment:
is it possible to have PA:LIST() here ?

Reply:
It is actually a List. I'll update the type here. Thanks